store/store.rb

```diff
@@ -0,0 +1,130 @@
+require 'json'
+require './classes/categories/book'
+
+def get_path(path)
+  data_hash = nil
+  if Dir.exist?('./store') && !File.zero?(path)
+    raw = File.open(path)
+    file = File.read(raw)
+    data_hash = JSON.parse(file) if file != ''
+  end
+  data_hash
+end
+
+def set_path(category, data)
+  File.write(category, nil, mode: 'w')
+  File.write(category, JSON.dump(data), mode: 'w')
+end
+
+# rubocop:disable all
+def save_data
+  if @books.size.positive?
+    books_json = items_2_json(@books)
+    set_path('./store/books.json', books_json) if books_json.size.positive?
+  end
+
+  if @games.size.positive?
+    game_json = items_2_json(@games)
+    set_path('./store/games.json', game_json) if game_json.size.positive?
+  end
+
+  if @movies.size.positive?
+    movie_json = items_2_json(@movies)
+    set_path('./store/movies.json', movie_json) if movie_json.size.positive?
+  end
+
+  if @music_albums.size.positive?
+    music_album_json = items_2_json(@music_albums)
+    set_path('./store/music_albums.json', music_album_json) if music_album_json.size.positive?
+  end
+end
+
```

Comment:
- [ ] Optional
If any of your methods are longer than ~5 lines, the code could likely be cleaner. You'll want to extract logic into a smaller helper method. This is not a general rule but is applicable to a very logical task like this.